manifest/panel/entrypoint.sh

```diff
@@ -66,10 +66,11 @@ function startServer {
     fi
 
 
-    # is mobile app enalbed
+    # CHecking is mobile app enalbed
```

Comment:
maybe replace with `Check if the mobile app is enabled`

Reply:
thanks